src/components/form/form.tsx

```diff
@@ -234,4 +244,35 @@ export class Form {
             };
         });
     }
+
+    private getCustomErrorMessages(originalErrors: AjvError[]): AjvError[] {
```

Comment:
Oooh, so nice! Both with the hiding of AjvError and the example you added 😍 👏👏👏

Reply:
Thanks a lot @adrianschmidt!